platforms/posix/cmake/sitl_target.cmake

```diff
@@ -37,6 +37,7 @@ ExternalProject_Add(sitl_gazebo
 	CMAKE_ARGS
 		-DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}
 		-DSEND_ODOMETRY_DATA=ON
+		-DGENERATE_ROS_MODELS=ON
```

Comment:
Shouldn't this be off?

Reply:
Default to ON makes more sense to me. Specially because we were already generating ROS (1) compatible sensors anyway (depth_camera uses the openni plugin, which is a gazebo_ros plugin), used by PX4/avoidance.

Reply:
Which default value is better? I think ON looks good since the ROS plugins are used as default sdf.